cmd/yace/scraper.go

```diff
@@ -38,16 +38,16 @@ type cachingFactory interface {
 	Clear()
 }
 
-func NewScraper(featureFlags []string) *scraper { //nolint:revive
```

Comment:
Is it worth making scraper public as it's used only in `main.go`?

Reply:
The linter was complaining about the lint being present.

Reply:
I thought I saw a nolintlint false-positive bug recently.

Reply:
That could be it.

Reply:
IIUC we could revert this specific change if it was just a false positive?